### DISCUSSION TITLE
Muting Host PC speakers

### DISCUSSION THREAD

**Discussion by jazzzeee** [LizardByte/community#156] [Sunshine]
Apologies for another potentially basic question.

Is it possible with Sunshine to mute the host PC speakers when streaming to Apple TV using Moonlight app? 

I can switch from the Nvidia sound card to the onboard CPU soundcard with my PC sound settings or pull the speaker plug and that mutes the host speakers. However, i'm wondering if there is a way to automatically mute the speakers through the Sunshine configuration? 

I'm not using steam or virtual audio sinks (i don't think). I'm just playing games from desktop. 

Thanks

**Comment by buthig666** [comment]
the audio-info.exe (sorry if that's not exactly what it's called) is supposed to give you the id of your audio devices.  Pick one that you never use.   Unfortunately, the exe opens and immediately minimizes for me.  So I dug around in device manager and found the same info.

**Reply by ReenigneArcher** [reply]
run the program from command prompt so you can see the output

**Reply by jazzzeee** [reply]
Thanks. I've put my Realtek for both audio and it works. The Nividia overlay interferes with it so that needed to be turned off. Hopefully that can be automated in a future upgrade (said by someone that knows little about computers).

**Reply by TheFurnace** [reply]
You need to run the exe from either Command Prompt or powershell to see the output.

[a comment by an automated account is not shown]
